packages/server/src/service/trpc.test.ts

```diff
@@ -0,0 +1,61 @@
+import { sampleStateInfo } from './letter'
+import { mocked } from 'ts-jest/utils'
+import { VbmRpc } from './trpc'
+import mailgun, { Mailgun } from 'mailgun-js'
+import twilio, { Twilio } from 'twilio'
+import { Request } from 'express'
+import { toContactMethod, StateInfo } from '../common'
+import { sendFaxes as unmockedSendFaxes } from './twilio'
+import { isE164 } from '../common/util'
+
+// To avoid doing actual requests (and setting a lot of new envs for tests),
+// we mock the majority of functions/modules related to trpc.
+
+jest.mock('mailgun-js')
+jest.mock('twilio')
+jest.mock('./twilio')
+jest.mock('./trpc')
+jest.mock('../common', () => {
+  const common = jest.requireActual('../common')
+  common.processEnvOrThrow = jest.fn().mockImplementation(key => {
```

Comment:
Instead of doing this, why don't you just set the values in the `test` environment to a stub 'xxx'?